infrastructure/monitoring/elastalert/rules/alert.yaml

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+name: 'alert'
```

Comment:
This file defines how an alert is detected from Elasticsearch and what to do with it. The alert rules themselves are defined in Kibana either through the UI or (preferable) in `infrastructure/monitoring/kibana/alert-rules.json`